src/ImageSharp/Formats/Tiff/PhotometricInterpretation/Rgba16161616TiffColor{TPixel}.cs

```diff
@@ -69,6 +83,12 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
                         pixelRow,
                         pixelRow.Length);
 
+                    if (hasAssociatedAlpha)
+                    {
+                        PixelOperations<TPixel>.Instance.ToVector4(this.configuration, pixelRow, vectorsSpan);
```

Comment:
Can't we use `PixelOperations<TPixel>.Instance.FromVector4Destructive` passing `PixelConversionModifiers.Premultiply` here? This allows us to use Avx2 unpremultiplication.